apps/fluent-tester/src/TestComponents/Shadow/ShadowButtonTestSection.tsx

```diff
@@ -14,25 +18,31 @@ const CustomFABShadow64 = FAB.customize({
 export const ShadowButtonTestSection: React.FunctionComponent = () => {
   const t = useFluentTheme();
 
-  if (Platform.OS === 'ios') {
-    return (
-      <View style={shadowTestPageStyles(t).backgroundColor}>
-        <FAB icon={iconProps} style={commonTestStyles.vmargin}>
-          FAB with default shadow
-        </FAB>
-        <CustomFABShadow64 icon={iconProps} style={commonTestStyles.vmargin}>
-          Custom FAB with shadow64
-        </CustomFABShadow64>
-        <CustomFABNoShadow icon={iconProps} style={commonTestStyles.vmargin}>
-          Custom FAB with no shadow
-        </CustomFABNoShadow>
-      </View>
-    );
-  } else {
-    return (
-      <View style={stackStyle}>
-        <Text>Shadows only implemented for iOS at this time</Text>
-      </View>
-    );
-  }
+  return (
+    <View style={shadowTestPageStyles(t).backgroundColor}>
+      {Platform.OS === 'ios' && (
+        <View style={stackStyle}>
+          <FAB icon={iconProps} style={styles.marginBetweenComponentsWithShadow}>
+            FAB with default shadow
+          </FAB>
+          <CustomFABShadow64 icon={iconProps} style={styles.marginBetweenComponentsWithShadow}>
+            Custom FAB with shadow64
+          </CustomFABShadow64>
+          <CustomFABNoShadow icon={iconProps} style={styles.marginBetweenComponentsWithShadow}>
+            Custom FAB with no shadow
+          </CustomFABNoShadow>
+        </View>
+      )}
+      {Platform.OS !== 'android' && (
```

Comment:
Could we make the platform checks a little more obvious? Maybe pull out to separate variables with names like `supportsFABShadow` or something? It's not clear to me what's special about Android and iOS to me in this test.

Reply:
Just updated